src/legacy/app/components/DeviceSelector.jsx

```diff
@@ -40,7 +10,10 @@ import { Iterable } from 'immutable';
 import { bool, func, instanceOf, number, oneOfType, string } from 'prop-types';
 
 import HotkeyedDropdown from '../../components/HotkeyedDropdown';
-import portPath from '../../portPath';
+
+const getTraits = traits => {
+    return Object.keys(traits).filter(key => traits[key]);
```

Comment:
Are any keys nullish? Why is it not enough with Object.keys()?

Reply:
Good question, I honestly cannot remember writing this ;) I think this was from the very early stages of updating, when nothing worked and I was just desperate to get something showing!

Reply:
They are either true or false, so Object.keys will not be enough as we only want to return the traits that are set to `true`.